### PR TITLE
[CDAP-14812] update styling of field level lineage operation modal

### PR DESCRIPTION
**Pull request by elfenheart**
JIRA: https://issues.cask.co/browse/CDAP-14812
Build: https://builds.cask.co/browse/CDAP-UDUT225

### CODE REVIEW

File: cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/ModalContent.js [hunk 1]
```diff
@@ -62,18 +62,16 @@ function ModalContentView({ operations, activeIndex }) {
 
   return (
     <div className="operations-container">
-      <Navigation />
+      <h5 className="summary-text">{T.translate(`${PREFIX}.summaryText`, { sources, targets })}</h5>
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/ModalContent.js]: nit: Can we use the `Heading` component?

File: cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/ModalContent.js [hunk 2]
```diff
@@ -62,18 +62,16 @@ function ModalContentView({ operations, activeIndex }) {
...
 
-      <div className="summary-text">
-        {T.translate(`${PREFIX}.summaryText`, { sources, targets })}
-      </div>
+      <Navigation />
 
-      <div className="last-execution">
+      <h6 className="last-executed">
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/components/FieldLevelLineage/OperationsModal/ModalContent.js]: Same here

File: cdap-ui/app/cdap/styles/common.scss [hunk 3]
```diff
@@ -192,6 +192,32 @@ body {
     }
   }
 
+  .grid.grid-container.grid-compact {
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/styles/common.scss:195]: Nice!!

File: cdap-ui/app/cdap/styles/common.scss [hunk 4]
```diff
@@ -192,6 +192,32 @@ body {
...
+    .grid-header,
+    .grid-body {
+      .grid-row > div {
+        padding: 5px 7px;
+        min-height: 29px;
```
Comment (ajainarayanan) [inline cdap-ui/app/cdap/styles/common.scss]: nit: Do we need this min-height?

**Review by ajainarayanan — COMMENTED** [review]
Minor comments.

File: cdap-ui/app/cdap/styles/common.scss (hunk 4 above)
Comment (elfenheart) [reply cdap-ui/app/cdap/styles/common.scss]: Yes we do, otherwise it defaults to 34px

**Review by ajainarayanan — APPROVED** [review]
👍 LGTM

**Pull request merged**